### DISCUSSION TITLE
TuringCourses/short_term/acm/

### DISCUSSION THREAD

**Discussion by giscus** [ZJU-Turing/TuringCourses#86] [General]
# TuringCourses/short_term/acm/

图灵班学习指南

https://zju-turing.github.io/TuringCourses/short_term/acm/

**Comment by Fairicle** [comment]
2023 年的短学期（个人赛阶段）结束了，进入预组队阶段的有 21 人，最终将会在这 21 人中选出 12 人作为校队。一些有趣的统计数据：这 21 人中 80% 的人来自浙江，个人赛前十名均由浙江同学包揽；21 人中 60% 的人拥有 NOI/WC/APIO 银牌及以上的奖项。所以如果想要入选校队，需要很强劲的实力。可能是高考花费时间较多的缘故，图灵同学在这方面都不太行，今年几位图灵同学在个人赛阶段均获得了四十名开外的好成绩（也就是垫底）。如果没有足够的实力，建议还是不要选择这门课，因为不能学到什么东西而且还会在比赛时坐牢。

**Reply by Fairicle** [reply]
以及事实证明，如果你水平不够，不仅会收获坐牢体验和垫底排名，还会因此收获很不理想的成绩。强烈建议选这门课的同学至少要达到 cf rating 2100 的水平。

**Comment by ChenZ01** [comment]
快……逃……
